code/modules/mob/living/carbon/human/species_types/jellypeople.dm

```diff
@@ -551,6 +551,12 @@
 	icon_icon = 'icons/mob/actions/actions_slime.dmi'
 	background_icon_state = "bg_alien"
 	var/activation_type = SLIME_ACTIVATE_MINOR
+	var/datum/species/jelly/luminescent/species
```

Comment:
this var doesnt seem to be used and even if it was you dont clean it up if the object its referencing gets qdel'd. what are you trying to do here?